packages/connect/src/createScClient/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./createScClient.js"
-export { healthChecker } from "./ScProvider/index.js"
```

Comment:
We don't want to export `ScProvider`. So, probably the easiest is to leave the `./ScProvider/index.js` untouched, and then from here do: `export * from "./ScProvider/Health.js"

Reply:
Thank you 🙏🏽

Reply:
No, thank **you**!  I was the one who forgot to export the types of the `healthChecker`.